gen3/docs/gen3-resources/developer-guide/microservices.md

```diff
@@ -5,7 +5,6 @@ Gen3 features and functionality are enabled by independent and modular microserv
 While the average user does not need to know the details and names of each microservice, if you are interested in adding new features or modifying your Gen3 system in some way it may be helpful to have a deeper understanding of a specific microservice.  We have included brief descriptions below along with a link to their documentation in GitHub.
 
 ## [Aggregated Metadata Service (AggMDS)][aggmds github]
```

Comment:
I still worry about having this separated from the MDS. At least we should put them together

Reply:
AggMDS was consolidated under MDS